docs/src/reference/api/cmfpackage.json/index.md

```diff
@@ -0,0 +1,199 @@
+
+### packageId
+
+The *most* important things in your cmfpackage.json are the packageId and version fields as they will be required. The packageId and version together form an identifier that is assumed to be completely unique. Changes to the package should come along with changes to the version.
+
+The packageId is automatically generated when the package is created via the [`cmf new`](../../commands/new.md) and by default contains the packageType (eg: *Cmf.Feature.Business*).
+
+### version
+
+The *most* important things in your cmfpackage.json are the packageId and version fields as they will be required. The packageId and version together form an identifier that is assumed to be completely unique. Changes to the package should come along with changes to the version.
+
+Version must be parseable by [node-semver](https://github.com/npm/node-semver).
+
+### description
+
+Put a description in it.  It's a string.
+
+### packageType
+
+The packageType is defined via an Enum, check all the valid values [here](packagetype.md).
+
+### isInstallable
+
+Boolean value *(default true)*. A value of true indicates that the package is prepared to be installed. This value is usually only false for packages of type Tests.
+
+### isUniqueInstall
+
+Boolean value *(default false)*. A value of true indicates that the package is only installed once per application server, a second run of this package doesn't re-install it. This value is usually only true for packages of type Data, IoTData and Database.
+
+### isToSetDefaultSteps
+
+Boolean value *(default true)*. A value of true indicates that a set of predefined steps (each packageType have a set of steps) will be used.
+This value is usually only false when some debug is needed or for some reason the default steps are not working as expected.
+
+### steps
+
+The `steps` field is an array of steps that will be executed during the package deployment. This by default empty, because each package type have a set of predefined steps.
+This field is very useful when you want to add steps to be executed after the predefined or when combined with the property `isToSetDefaultSteps` as *false* you can override all the predefined steps.
+
+Example:
+```json
+{
+    "packageId": "Cmf.Custom.Data",
+    "version": "1.1.0",
+    "description": "Cmf Custom Data Package",
+    "packageType": "Data",
+    "isInstallable": true,
+    "isUniqueInstall": true,
+    "steps": [
+    {
+        "type":"Generic",
+        "onExecute":"scriptToRun.ps1"
+    }]
+}
+```
+
+### contentToPack
+
+The `contentToPack` field is an array of file patterns (source and target, check below) that describes the files and folders to be included when your package is packed.
+File patterns follow a similar syntax to .gitignore, but reversed: including a file, directory, or glob pattern (*, **/*, and such) will make it so that file is included in the zip when it's packed.
```

Comment:
you'll need to escape the asterisks, otherwise they will be interpreted as markdown

Reply:
good catch